src/interfaces/docker_interface.py

```diff
@@ -82,23 +83,27 @@ def up(self):
             )
 
     @bubble_exception_str
-    def down(self):
-        command = ["docker", "compose", "down"]
-        try:
-            with subprocess.Popen(
-                command,
-                cwd=str(self.config_dir),
-                stdout=subprocess.PIPE,
-                stderr=subprocess.STDOUT,
-            ) as process:
-                logger.debug(f"Running docker compose down with PID {process.pid}")
-                if process.stdout:
-                    for line in process.stdout:
-                        logger.info(line.decode().rstrip())
-        except Exception as e:
-            logger.error(
-                f"An error occurred while executing `{' '.join(command)}` in {self.config_dir}: {e}"
-            )
+    def down(self, persist):
+        if persist:
+            commands = [shlex.split("docker compose stop"), shlex.split("docker compose rm --force")]
```

Comment:
so, wait. if `persist` is `True` then we execute `rm --force` ?

Reply:
Also, do we really need another module import just to convert a string with spaces to a list? :-P